### ISSUE TITLE
JS redirection into default DW constructor

### ISSUE THREAD

**Issue by Dr-Yukon** [dwp-forge/batchedit#33]
Please look here https://github.com/splitbrain/dokuwiki/pull/2958#pullrequestreview-345025707

When browser load JS without defer or async attribute, it stopped page parsing as a default. https://hsto.org/files/fc6/581/de4/fc6581de40d249e5905cf3fa238286eb.png

Async attribute uses for JS base libraries (jQuery etc.) asynchronised downloading from external CDN and they start working immediately after they answer to browser pings. https://hsto.org/files/a07/823/bae/a07823bae90b4612bc5e4933674b34ce.png

Defer attribute uses for JS scripts, based on libraries, they start working only after all downloads and parses. It means that libraries start works before scripts. https://hsto.org/files/378/af7/eff/378af7eff5e34d89b75584d626c48cae.png

**This small feature needed for DW sites fastening**. 

But BatchEdit uses loading of self small and fast scripts with jQuery instructions which are separate from DW's default /js.php script combiner and tries to work before async'ed jQuery library starting. **All brokeh**

Please fix BatchEdit ASAP, please, please!

**Comment by Dr-Yukon** [comment]
wow. they already merged it in https://github.com/splitbrain/dokuwiki/pull/2974

**Comment by dwp-forge** [comment]
I pushed the fix (8aee261), but I will postpone the "official" release until DW releases. Who know what else they will break this time.

**Comment by Dr-Yukon** [comment]
nice, it works
